### ISSUE TITLE
Cross-platform API not working on Android

### ISSUE THREAD

**Issue by havart-netcompany** [xamarin/xamarin.exposurenotification#36]
When running this repository's sample app on Android (with a whitelisted Google account and a "checked in" Google Play Services beta) after removing the line:

> Xamarin.ExposureNotifications.ExposureNotification.OverrideNativeImplementation(new Services.TestNativeImplementation());

we get:

> Android.Gms.Common.Apis.ApiException: 6:

when calling:

> IExposureNotificationClient.StartAsync()

The Api Exception code 6 means RESOLUTION_REQUIRED: https://developers.google.com/android/reference/com/google/android/gms/common/api/CommonStatusCodes?hl=de#public-static-final-int-resolution_required. Google's official sample app gets these API exceptions, then handles them by calling

> apiException.getStatus().startResolutionForResult(this, RequestCodes.REQUEST_CODE_START_EXPOSURE_NOTIFICATION);

This seems to be missing from the cross-platform Xamarin Exposure Notification API. Is this about to be fixed soon, is it the App code that's missing something to handle this, or is there some required setting I am missing to make this work?

**Comment by mattleibow** [comment]
Just a note:

There are 2 codes

```java
  public static final int REQUEST_CODE_START_EXPOSURE_NOTIFICATION = 1111;
  public static final int REQUEST_CODE_GET_TEMP_EXPOSURE_KEY_HISTORY = 2222;
```

**Comment by Redth** [comment]
Yeah we'll look up what the other is used for now.

In the meantime, fixing the start API exception by handling it through the intent.

**Comment by Redth** [comment]
@havart-netcompany we are looking at this in https://github.com/xamarin/xamarin.exposurenotification/pull/38

**Comment by Redth** [comment]
@havart-netcompany I believe master is now up to date with the changes required and released to nuget as 0.3.0-preview.

Please open a new issue if there are any more concerns, thanks!
